docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,177 @@
+* Many people with mobility impairments use custom keyboards or keyboard emulators.
+* Some people rely on speech to interact with web applications and need to be able to address interactive elements.
+* Someone who cannot hear or hear well requires visible text captions for any audible content or an equivalent visual cue.
+* Users may customize their operating system, browser or web page display properties to make web applications easier for them to use (like reversing contrast, increasing font size, or removing images). 
+
+These are just a few core concepts developers need to keep in mind when developing user interfaces that work for everyone. More information is available from the W3C's Web Accessibility Initiative's `How People with Disabilities Use the Web: Overview <http://www.w3.org/WAI/intro/people-use-web/Overview.html>`_.
+
+Use semantic markup
+*******************
+
+The role, state and associated properties of an element are exposed to users of Assistive Technology either directly through the DOM or through the Accessibility API. Using elements for purposes other than the ones they are intended for have the consequence of falsely reporting the role, state and associated properties of the element to these users. This breaks features designed to make web apps easier to use and can often result in confusion when expected behaviors are not available.
+
+If the semantics and behavior you need already exist in a native HTML5 element, you should use that element:  
+
+* If you want a button, use the ``<button>`` element and not a ``<div>`` that looks and behaves like a button. 
+* If you want a checkbox, you should use an ``<input type=checkbox>`` and not try to recreate the states and properties you get with the native element for free. Chances are, you will not fully replicate all of them, i.e. making it focusable, toggling its checked state upon ``space`` or ``enter`` keypresses, exposing its label and  "`checkedness <http://www.w3.org/TR/html5/forms.html#concept-fe-checked>`_" to the Accessibility API (did you know a checkbox can be in an intermediate state?)
+
+Don't use an element because of its default style or because it provides a convenient styling hook:
+
+* Are you choosing the Heading level (``<h1>`` - ``h6>``) because of its physical size or appearance?  Heading levels should be determined by their logical order in the hierarchical structure of the document.
+* Are you really marking up a list of items, or are you using an ``<ul>`` as a styling hook?  Unordered lists should only be used when marking up a collection of related items.  Ordered lists are for a collection of related items where their order in the list is of importance.  Screen readers provide extra feedback and functionality for lists and other elements with semantic importance.  It can confusing or cumbersome when this feedback is inaccurately reported.
```

Comment:
"Use ordered and unordered lists (``<ol>`` and ``<ul>``, respectively) only when you are marking up a collection of related items. Do not use unordered lists for styling purposes. Screen readers provide extra feedback..."